DESCRIPTION

```diff
@@ -0,0 +1,18 @@
+Package: vitalsigns
+Title: A Utility Package for Conservation International's Vital Signs Project 
+Version: 0.1
+Authors@R: person("Thomas", "Johnson", email = "thomascjohnson@gmail.com",
```

Comment:
I can't find any info on how to include these. Reference: http://r-pkgs.had.co.nz/description.html

Wickham doesn't mention anything about a date field, but does mention that maintainer is designated by "cre" in the Authors field.

Would you mind helping me out here?